magento/graphql-ce
# [Wishlist coverage] Add items to wishlist

### Description (*)
This PR adds a mutation to the WIshlistGraphQl module functionality for adding items to the wishlist.

### Fixed Issues (if relevant)
#59 : [Mutations] My Account > Wish List

### Manual testing scenarios (*)
1. Create a product with sku "simple_product"
2. You need to be logged in, so use the corresponding bearer authentication token in the GraphQL request headers
3. Use the following mutation for adding product to the wishlit:

```graphql
mutation {
  addItemsToWishlist(input: {sku_list: ["simple_product"]}) {
    wishlist {
      items_count
      name
      items {
        product {
          name
        }
      }
    }
  }
```
4. You should see the information about the wishlist including the recently added product
